app/app.py

```diff
@@ -35,11 +36,10 @@ def questions():
 
     return question_dict, status
 
-@app.errorhandler(QuestionNotFound)
-def question_not_found(error):
-
-    return error.json, 404
+@app.errorhandler(HttpError)
+def handle_http_error(error):
 
+    return error.json, error.json.get("status", 500)
```

Comment:
Should the status be part of the object rather than JSON?

Reply:
well it is part of the object, but wanted to keep in line with the rest of the functions returning `body, status`